### DISCUSSION TITLE
android pagview动画播放闪烁

### DISCUSSION THREAD

**Discussion by maleah714** [Tencent/libpag#1959] [Q&A]
【版本信息】
com.tencent.tav:libpag:4.3.33

【平台信息】
车机android系统
Android版本：Android 10

【预期的表现】
动画循环播放时不闪烁

demo工程、复现视频及logcat日志在下面的链接中：
链接: https://pan.baidu.com/s/1rmUeP1Xpn3u2PXnZifWC6w 提取码: 7633

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.3.33。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
